apps/desktop/src/index.html

```
@@ -1369,25 +1369,25 @@ <h3 id="port-config-title" class="text-xl font-light text-[var(--text-primary)]
                 <div>
                     <label class="block text-2xs text-[var(--text-muted)] uppercase tracking-wider mb-2" data-i18n="mixedPort">Mixed Port</label>
                     <input id="port-mixed-input" type="number" min="0" max="65535" class="form-control form-control-md" placeholder="7890">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)